### PR TITLE
Moving this repo to teamhephy org

### PR DESCRIPTION
**Pull request by Cryptophobia**
Moving this Go repo to teamhephy org and resolving glide up dependencies.

### CODE REVIEW

File: appsettings/appsettings_test.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ const appSettingsFixture string = `
     "routable": true,
     "whitelist": ["1.2.3.4", "0.0.0.0/0"],
     "autoscale": {"cmd": {"min": 3, "max": 8, "cpu_percent": 40}},
-    "label": {"git_repo": "https://github.com/deisthree/controller-sdk-go", "team" : "deis"},
+    "label": {"git_repo": "https://github.com/teamhephy/controller-sdk-go", "team" : "deis"},
```
Comment (kingdonb) [inline appsettings/appsettings_test.go:23]: team: "deis"

Not sure what this does...

File: appsettings/appsettings_test.go [hunk 2]
```diff
@@ -35,15 +35,15 @@ const appSettingsUnsetFixture string = `
...
```
Comment (kingdonb) [inline appsettings/appsettings_test.go:38]: Same

File: appsettings/appsettings_test.go [hunk 3]
```diff
@@ -35,15 +35,15 @@ const appSettingsUnsetFixture string = `
...
     "created": "2014-01-01T00:00:00UTC",
     "updated": "2014-01-01T00:00:00UTC",
     "uuid": "de1bf5b5-4a72-4f94-a10c-d2a3741cdf75"
 }
 `
 
-const appSettingsSetExpected string = `{"maintenance":true,"routable":true,"whitelist":["1.2.3.4","0.0.0.0/0"],"autoscale":{"cmd":{"min":3,"max":8,"cpu_percent":40}},"label":{"git_repo":"https://github.com/deisthree/controller-sdk-go","team":"deis"}}`
-const appSettingsUnsetExpected string = `{"maintenance":true,"routable":true,"whitelist":["1.2.3.4","0.0.0.0/0"],"autoscale":{"cmd":{"min":3,"max":8,"cpu_percent":40}},"label":{"git_repo":"https://github.com/deisthree/controller-sdk-go","team":"deis"}}`
+const appSettingsSetExpected string = `{"maintenance":true,"routable":true,"whitelist":["1.2.3.4","0.0.0.0/0"],"autoscale":{"cmd":{"min":3,"max":8,"cpu_percent":40}},"label":{"git_repo":"https://github.com/teamhephy/controller-sdk-go","team":"deis"}}`
+const appSettingsUnsetExpected string = `{"maintenance":true,"routable":true,"whitelist":["1.2.3.4","0.0.0.0/0"],"autoscale":{"cmd":{"min":3,"max":8,"cpu_percent":40}},"label":{"git_repo":"https://github.com/teamhephy/controller-sdk-go","team":"deis"}}`
```
Comment (kingdonb) [inline appsettings/appsettings_test.go:46]: Same

**Review by kingdonb — APPROVED** [review]

File: appsettings/appsettings_test.go (hunk 1 above)
Comment (Cryptophobia) [reply appsettings/appsettings_test.go:23]: It's just a field on the labels of the appsettings route on controller API. Test passes so we can keep this label the same for now. The controller route it is supposed to mimic is `v2/apps/example-go/settings`

File: appsettings/appsettings_test.go (hunk 2 above)
Comment (Cryptophobia) [reply appsettings/appsettings_test.go:38]: Same as above.

File: appsettings/appsettings_test.go (hunk 3 above)
Comment (Cryptophobia) [reply appsettings/appsettings_test.go:46]: Same as above.

**Comment by Cryptophobia** [comment]
Thanks for review @kingdonb !

**Comment by Cryptophobia** [comment]
Merging now and will test one more time! 

Looks like this closes https://github.com/teamhephy/controller-sdk-go/pull/1 for now. We should look at moving from quay.io/deisci to quay.io/hephyci for the jenkins charts?

**Comment by Cryptophobia** [comment]
All tests pass and the container builds in the teamhephy `/src` folder in $GOPATH. We are good to go for now.

**Pull request merged**